integrations/tensorflow/iree-dialects/include/iree-dialects/Dialect/LinalgExt/IR/LinalgExtOps.td

```diff
@@ -75,12 +75,14 @@ def IREELinalgExt_ScatterOp : IREELinalgExt_Op<"scatter",
   }];
   let arguments = (ins
       Variadic<AnyRankedTensorOrMemRefType>:$inputs,
-      Variadic<AnyRankedTensorOrMemRefType>:$outputs
+      Variadic<AnyRankedTensorOrMemRefType>:$outputs,
+      DefaultValuedAttr<BoolAttr, "false">:$unique_indices
   );
   let results = (outs Variadic<AnyRankedTensor>:$results);
   let regions = (region AnyRegion:$region);
   let assemblyFormat = [{
-    attr-dict (`ins` `(` $inputs^ `:` type($inputs) `)`)?
+    `unique_indices` `(` $unique_indices `)`
```

Comment:
Could you make this 

```
attr-dict  (`unique_indices` `(` $unique_indices `)`)?
(`ins` `(` $inputs^ `:` `type($inputs) `)`)? 
`outs` `(` $outputs `:` `type($outputs) `)`
$region (`-> ` type($results)^)?
```

Reply:
I was thinking the same thing (which looks more reasonable to me), but I found that we use this style for sort op. I'll update this.

Maybe we also want to update sort op to follow this style? If so, I can make another PR for it.

Reply:
Sounds good.